bears/general/CPDBear.py

```diff
@@ -0,0 +1,41 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
```

Comment:
I'm not sure how it works, but shouldn't it check all the files at once to detect duplicated code from several files? Say, if one file almost the same as another one.